### PR TITLE
Update free-courses-en.md (Changes Made)

### PR DESCRIPTION
**Pull request by HarshRanaOC**
Included an excellent course on the Django Python framework.

## What does this PR do?
Add resource(s) 

### Why is this valuable (or not)?
I have tryed this course

### How do we know it's really free?
It's on youtube.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by HarshRanaOC** [comment]
Hello @sohansai, I've followed all the contribution guidelines and crafted a flawless pull request.

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1433,6 +1433,7 @@
 * [Django Wednesdays](https://www.youtube.com/playlist?list=PLCC34OHNcOtqW9BJmgQPPzUpJ8hl49AGy) - Codemy.com
 * [Python Django Tutorial 2018 for Beginners](https://www.youtube.com/playlist?list=PL-J2q3Ga50oOpni_xS2PPUe4mf9lM96dD) - Clever Programmer
 * [Python Django Tutorial 2021](https://www.youtube.com/playlist?list=PL-51WBLyFTg1pUMaTJ4WSgnyvWfLGmwDm) - Dennis Ivy
+* [Python Django Web Framework - Full Course for Beginners](https://www.youtube.com/watch?v=F5mRW0jo-U4) - freeCodeCamp
```
Comment (Thenlie) [inline courses/free-courses-en.md:1436]: For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add ‘changes made’ to the title and reviewer will take another look as soon as they are available. Thanks!

**Review by Thenlie — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (Thenlie) [reply courses/free-courses-en.md:1436]: Whoops, Eric's comment wasnt there when I started 😅

Comment (HarshRanaOC) [reply courses/free-courses-en.md:1436]: Hello @Thenlie, the course consists of a single video that lasts for 4 hours. The creator hasn't organized it into a playlist.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**